Sources/RemoteMessaging/Matchers/UserAttributeMatcher.swift

```diff
@@ -136,6 +154,22 @@ public struct UserAttributeMatcher: AttributeMatcher {
             } else {
                 return RangeIntMatchingAttribute(min: matchingAttribute.min, max: matchingAttribute.max).matches(value: privacyProDaysUntilExpiry)
             }
+        case let matchingAttribute as PrivacyProPurchasePlatformMatchingAttribute:
+            return StringArrayMatchingAttribute(matchingAttribute.value).matches(value: privacyProPurchasePlatform ?? "")
+        case let matchingAttribute as PrivacyProSubscriptionStatusMatchingAttribute:
+            guard let value = matchingAttribute.value else {
+                return .fail
+            }
+
+            guard let status = PrivacyProSubscriptionStatus(rawValue: value) else {
+                return .fail
+            }
```

Comment:
Deliberately fail if we provide a status string that doesn't match any supported value.